src/jit/morph.cpp

```diff
@@ -2598,6 +2582,13 @@ GenTreeCall* Compiler::fgMorphArgs(GenTreeCall* callNode)
     bool            callIsVararg      = call->IsVarargs();
 #endif
 
+#ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
+    // If fgMakeOutgoingStructArgCopy is called and
+    // copies are generated, hasStackArgCopy is set 
+    // to make sure to call evalArgsToTemp.
```

Comment:
>evalArgsToTemp [](start = 28, length = 14)

Since you're referring to a function, can you make sure the name matches exactly, i.e., EvalArgsToTemps()